IceCubesApp/App/IceCubesApp-release.entitlements

```diff
@@ -2,22 +2,10 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
-	<key>aps-environment</key>
```

Comment:
This should probably not be commited

Reply:
Arrgggh. And here I am trusting XCode with Git commands. Sorry.